apps/rush-lib/src/data/RushConfiguration.ts

```diff
@@ -173,14 +173,29 @@ export default class RushConfiguration {
 
     // If the version is missing or malformed, fall through and let the schema handle it.
     if (expectedRushVersion && semver.valid(expectedRushVersion)) {
-      if (semver.lt(Rush.version, expectedRushVersion)) {
-        throw new Error(`Unable to load ${rushJsonBaseName} because its RushVersion is`
-          + ` ${rushConfigurationJson.rushVersion}, whereas @microsoft/rush-lib is version ${Rush.version}.`
-          + ` Consider upgrading the library.`);
-      } else if (semver.lt(expectedRushVersion, MINIMUM_SUPPORTED_RUSH_JSON_VERSION)) {
+      // Make sure the requested version isn't too old
+      if (semver.lt(expectedRushVersion, MINIMUM_SUPPORTED_RUSH_JSON_VERSION)) {
         throw new Error(`${rushJsonBaseName} is version ${expectedRushVersion}, which is too old for this tool. ` +
           `The minimum supported version is ${MINIMUM_SUPPORTED_RUSH_JSON_VERSION}.`);
       }
+console.log('whoopsie: ' + Rush.version + ' vs ' + expectedRushVersion);
```

Comment:
>whoopsie [](start = 13, length = 8)

?

Reply:
oops, that was some debug code

Reply:
oops, that was some debug code

---
In reply to: [183514399](https://github.com/Microsoft/web-build-tools/pull/618#discussion_r183514399) [](ancestors = 183514399)